projects/lights_control/inc/lights_gpio.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+#include "event_queue.h"
+
+#include "status.h"
+
+typedef enum { LIGHTS_BOARD_FRONT, LIGHTS_BOARD_REAR, NUM_LIGHTS_BOARDS } LightsBoard;
+
+// used for figuring out which board we're operating on is called before lights_gpio_init since
+// lights_gpio_init needs to know which board it needs to initialize
+StatusCode lights_gpio_get_lights_board(LightsBoard *board);
```

Comment:
Why don't you incorporate this into your init?

Reply:
changed it to `lights_gpio_get_board`, and I moved the body of the function to init, where I save the type of board to `s_board`.

Other modules will need to know which board they're operating on, so this module will provide it for them.

Reply:
We generally try to avoid static variables within modules. I don't think there's any need for this module to even be aware of which board it is. If you pass in a config struct, there's no need to explicitly state it. The same goes for the rest of your modules.